partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,156 @@
+        from partcad.globals import init
+
+        try:
+            if p is None:
+                ctx.obj = init()
+            else:
+                ctx.obj = init(p)
+        except (yaml.parser.ParserError, yaml.scanner.ScannerError) as e:
+            exc = click.BadParameter("Invalid configuration file", ctx=ctx, param=p, param_hint=None)
+            exc.exit_code = 2
+            raise exc
+        except Exception as e:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)